ftw/file/content/dxfile.py

```diff
@@ -41,6 +41,11 @@ def __init__(self, data='', contentType='', filename=None):
             if mimetypeitem:
                 contentType = mimetypeitem.normalized()
 
+        # Products.MimetypesRegistry 2.0.8 returned unicode afterwards it was utf-8
+        # The field contentType, however requires utf-8, since its a zope.schema.BytesLine
```

Comment:
Maybe this reads better?

contentType (a zope.schema.BytesLine) requires utf-8, but Products.MimetypesRegistry <= 2.0.8 returns unicode rather than utf-8

Reply:
yeah, I just changed it, thanks